docs/superpowers/plans/2026-07-28-voice-chat-m1-proximity.md

```
@@ -0,0 +1,2747 @@
+   vendors/mafianet/Source/src/RakVoice.cpp
+cp /Users/enguerrand/dev/mafiahub/MafiaNet/Source/include/mafianet/RakVoice.h \
+   vendors/mafianet/Source/include/mafianet/RakVoice.h
+```
+
+`vendors/mafianet/CMakeLists.txt` globs `Source/src/*.cpp`, so no source list edit is needed.
+
+- [ ] **Step 2: Record the re-vendoring**
+
+Append to `vendors/mafianet/VERSION.txt`:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)